OpInfo: Use yield consistently instead of appending to lists

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #84567
* __->__ #84454
* #86324
* #86231

The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

peterbell10 committed:
OpInfo: Use yield consistently instead of appending to lists

The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

Lezcano commented:
Incredible effort! It would be good to add a test to make sure that people keep using this pattern rather than the list pattern.

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Rebase on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Rebase on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]

peterbell10 committed:
Update on "OpInfo: Use yield consistently instead of appending to lists"


The yield syntax reduces the numbers of tensors held in-memory when
running opinfo tests and also avoids constructing all samples when
only the first sample is going to be used.

[ghstack-poisoned]
